src/mixer/playermanager.h

```diff
@@ -131,6 +145,23 @@ class PlayerManager : public QObject, public PlayerManagerInterface {
         return QString("[PreviewDeck%1]").arg(i+1);
     }
 
+    // Returns the group for the ith Microphone where i is zero indexed
+    static QString groupForMicrophone(int i) {
+        // Before Mixxx had multiple microphone support the first microphone had
+        // the group [Microphone]. For backwards compatibility we keep it that
+        // way.
+        QString group("[Microphone]");
+        if (i > 0) {
+            group = QString("[Microphone%1]").arg(i + 1);
```

Comment:
That would have been nice to do in 2.0.x so that we could start migrating everything to [Microphone1] and still be backwards compatible with 2.0.x :-/. But it's a good idea. I think I'd like to do it in a future PR since it would involve changes to src/engine which this PR isn't touching currently.